shared/chat/conversation/container.js

```diff
@@ -14,27 +15,46 @@ import type {Props} from '.'
 type OwnProps = {}
 type State = {
   sidePanelOpen: boolean,
+  inputText: string,
 }
 
 class ConversationContainer extends Component<void, Props, State> {
   state: State
+  _inputTextMap: {[key: string]: ?string}= {}
```

Comment:
keeping the input state in here for now so if you switch conversations its store but outside of redux. TODO make this into the route state